messages/accesstoken.store.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Authorize an org using an Salesforce Access Token\nWhen run interactively, the command prompts you for the access token.\nTo use the command in a CI/CD script, set the SFDX_ACCESS_TOKEN environment variable to the access token.",
```

Comment:
Small edit to the first sentence:

"authorize an org using a Salesforce access token"